website/content/en/v1.0/reference/settings.md

```diff
@@ -39,6 +39,10 @@ Karpenter surfaces environment variables and CLI parameters to allow you to conf
 
 [comment]: <> (end docs generated content from hack/docs/configuration_gen_docs.go)
 
+{{% alert title="Warning" color="warning" %}}
+If you've enabled [Security Groups per Pod](https://aws.github.io/aws-eks-best-practices/networking/sgpp/), one of the instance's ENIs is reserved. To avoid discrepancies between the `maxPods` value and the node's supported pod density, you need to set `RESERVED_ENIS=1`.
+{{% /alert %}}
+
```

Comment:
I think in settings, it's fine to just update the description in the table above. Right now we call out custom networking as the common use-case, adding SGPP should cover us. I do think this would be a good note to include in the [scheduling section on SGPP though](https://karpenter.sh/v1.0/concepts/scheduling/#pod-eni-resources-security-groups-for-pods). You could then link to the settings from there.